packages/slice-machine/server/src/api/services/getEnv.ts

```diff
@@ -50,7 +50,7 @@ function extractRepo(parsedRepo: ParseResult): string | undefined {
         return parsedRepo.subDomains[0];
       }
     default:
-      return;
+      return "";
```

Comment:
It's on the todo list to verify the sm.json, atm in order to reach this case they will have to modify the `sm.json` at run time. 

The main issue here is that the repo-name can't be undefined.

Reply:
Ahhh right I see what you mean now.
When Slice-Machine starts, we are Ok because we do validate the SM.json.
But if a change happens while Slice Machine is running, then we can't just throw an error yet because we don't handle them atm.

Thanks 👍